src/pages/About/About.css

```diff
@@ -0,0 +1,29 @@
+.about {
+  padding: 20px 50px;
+}
+
+.team {
+  display: flex;
+  justify-content: flex-start;
```

Comment:
Do you really need this flex-start property?

Reply:
Removed.